app/Http/Controllers/WordController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use App\Models\Word;
+use Illuminate\Http\Request;
+
+class WordController extends Controller
+{
+
+    public function getUserWords(Request $request, int $user_id)
+    {
+        $user = User::find($user_id);
+        if (!$user)
+        {
+            return response("User $user_id not found", 404);
+        }
+
+        $words = Word::where('user_id', $user_id)->get();
```

Comment:
rather than adding a property to the User model and then querying a user's words through that, we can just use the `words()` relation on the User class since we already have the `$user` anyway, like `$words = $user->words()->get();`